rpcs3/Emu/Cell/lv2/sys_game.cpp

```diff
@@ -15,11 +16,18 @@ error_code _sys_game_board_storage_read(vm::ptr<u8> buffer, u8 code)
 		return CELL_EFAULT;
 	}
 
+	u8 psid[16];
+	memset(psid, g_cfg.sys.console_psid_high << 8 | g_cfg.sys.console_psid_low, 16);
```

Comment:
```suggestion
	memset(psid, u128{g_cfg.sys.console_psid_high} << 64 | g_cfg.sys.console_psid_low, 16);
```

Reply:
E0289: No instance of constructor "u128::u128" matches the argument list
E0349 no operator "|" matches these operands

Reply:
This may be a misunderstanding on my side, but doesn't memset reinterprets the second argument as an unsigned char?

Reply:
`memset(psid, g_cfg.sys.console_psid_high << 8 | g_cfg.sys.console_psid_low, 16);` does work.

Reply:
What I mean is maybe it compiles, but have actually tried to set values with it? memcpy reinterprets the second argument as unsigned byte (am I missing sonething here?) so I assume high value would be completely ignored and only the lowest byte of low value be used abd repeat over 16 bytes.

Reply:
Reverted.

Reply:
> `memcpy(psid, &+g_cfg.sys.console_psid_low, 8);`
> `memcpy(psid + 8, &+g_cfg.sys.console_psid_high, 8);`

@Nekotekina "E0158: Expression must be an lvalue or a function designator"